src/sentry/api/endpoints/organization_auditlogs.py

```diff
@@ -1,27 +1,46 @@
+from rest_framework import serializers
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.bases import OrganizationEndpoint
 from sentry.api.bases.organization import OrganizationAuditPermission
 from sentry.api.paginator import DateTimePaginator
 from sentry.api.serializers import serialize
+from sentry.db.models.fields.bounded import BoundedIntegerField
 from sentry.models import AuditLogEntry
 
 EVENT_REVERSE_MAP = {v: k for k, v in AuditLogEntry._meta.get_field("event").choices}
 
 
+class AuditLogQueryParamSerializer(serializers.Serializer):
+
+    event = serializers.CharField(required=False)
+    actor = serializers.IntegerField(required=False, max_value=BoundedIntegerField.MAX_VALUE)
```

Comment:
it seems like this could be a useful field to have elsewhere since we can 500 if validation isn't there on id input.